### PR TITLE
Align with cookiecutter-ntc nautobot-plugin-v1.0

### LINKED ISSUE #18: Update https://github.com/nautobot/nautobot-chatops-cookiecutter to `nautobot-plugin-v1.0`

### PR DESCRIPTION
**Pull request by snaselj**
# Closes #18 

## Done

- Aligned with cookiecutter-ntc nautobot-plugin-v1.0
- Removed example

### CODE REVIEW

**Comment by snaselj** [comment]
This PR is ready for review now.

File: README.md [hunk 1]
```diff
@@ -21,57 +22,71 @@ Below are the steps outlined in detail for getting started along with various ti
...
+Choose from 1, 2 [1]:
+docs_base_url [https://docs.nautobot.com]:
+docs_app_url [https://docs.nautobot.com/projects/chatops-my-plugin/en/latest]: https://docs.nautobot.com/projects/chatops-my-plugin/en/latest
 Select setup_local_mattermost_dev_env:
 1 - Yes
 2 - No
 Choose from 1, 2 [1]: 
 
-Congratulations!  Your cookie has now been baked.
+Congratulations!  Your cookie has now been baked. It is located at /vagrant/nautobot-plugin-chatops-my-plugin.
```
Comment (jvanderaa) [inline README.md]:
```suggestion
Congratulations!  Your cookie has now been baked. It is located at /vagrant/nautobot-plugin-chatops-my-plugin
```

Comment (jvanderaa) [reply README.md]: Removes the period to help with double-click, copy and paste.

File: README.md [hunk 2]
```diff
@@ -21,57 +22,71 @@ Below are the steps outlined in detail for getting started along with various ti
...
 
 ⚠️⚠️ Before you start using your cookie you must run the following commands inside your cookie:
 
-* cp development/creds.example.env development/creds.env
 * poetry lock
+* cp development/creds.example.env development/creds.env
```
Comment (jvanderaa) [inline README.md:89]: Maybe worth a `cd {{ directory_created_in }}`

**Review by jvanderaa — APPROVED** [review]

**Comment by jvanderaa** [comment]
One minor change @snaselj

File: README.md (hunk 1 above)
Comment (snaselj) [reply README.md]: Fixed by moving the path to the next line, however this goes against this PR target, to be in sync with the cookiecutter-ntc plugin. Will introduce this change there as well.

File: README.md (hunk 2 above)
Comment (snaselj) [reply README.md:89]: It's mentioned above in the README, to run these commands inside the cookie. And as in the previous change, this PR should sync this template with cookiecutter-ntc.

**Pull request closed without merging**